testsV2/ut_repyv2api_openfileconsumesfilehandles.py

```diff
@@ -3,15 +3,14 @@
 """
 
 #pragma repy
+filehandles = []
 
-MAX_FILES = 5
 
-files = listfiles()
-filehandles = [None, None, None, None, None]
+RAND_FILE = "random.files.to.create.number"
 
 # Open random files
-for x in xrange(MAX_FILES):
-  filehandles[x] = openfile(files[x], False)
+for x in xrange(getresources()[0]['filesopened']):
+  filehandles.append(openfile((RAND_FILE + str(x + 1)), True)) # the plus 1 is for getting to 250 files!
```

Comment:
I see what your comment is trying to say, but 250 is an arbitrary limit, courtesy of the current version of the `restrictions.default` file. We are using `getresources` precisely so that we don't have to hardcode this number. 250 is "as many files as the resource restrictions allow", if that helps you reword the comment. (Minor nitpick: The comment should live on a line of its own, so as to limit the line length to 80 characters.)

BTW, I would suggest creating the list of file names separately from creating the actual files. This comes in handy when removing the files that the test creates, later on. (See @XuefengHuang's comment for reference.)